datasets/crass/README.md

```diff
@@ -0,0 +1,171 @@
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://www.crass.ai
+- **Repository:** https://github.com/apergo-ai/CRASS-data-set
+- **Paper:** [Needs More Information]
+- **Leaderboard:** https://www.crass.ai/ledger.php
+- **Point of Contact:** j.frohberg@apergo.net
+
+### Dataset Summary
+
+The data consists of so-called PCTs (Premise-Counterfactual Tuples). They contrast a hypothetical situation using a counterfactual conditional against a base premise. In the fixed target mode there is one correct answer and two alternative incorrect answers.
+
+### Supported Tasks and Leaderboards
+
+zero-shot-classification: The dataset can be used to benchmark a model for zero-shot or few-shot classification tasks, which consists in selecting the correct answer out of three possible answers. Success on this task is typically measured by achieving a high T1 accuracy. A [BART-based model] (https://huggingface.co/facebook/bart-large-mnli) model trained on MNLI currently achieves a [T1 accuracy of 0.48]. GPT-3 family accessed via OpenAI beta API using davinchi model currently achieves a [T1 accuracy of 0.56].
+
+### Languages
+
+en-US
+
+## Dataset Structure
+
+### Data Instances
+
+  {
+    "PCTID": 6,
+    "BatchID": 1,
+    "Premise": "A woman sees a fire.",
+    "QCC": "What would have happened if the woman had touched the fire?",
+    "CorrectAnswer": "She would have been burned.",
+    "Answer1": "She would not have been burned.",
+    "Answer2": "That is not possible.",
+    "PossibleAnswer3": "She would have seen fire.",
+    "canary_string": ""
+}
+
+### Data Fields
+
+PCTID: The ID of the item
+BatchID: A ID referencing creation process. Can be ignored.
+Premise: The description of the base premise that is the reference event against which the hypothetical event is compared.
+QCC: The so called questionized counterfactual conditional is a question creating a hypothetical event and contrasting it in a query-styled natural language form to the base premise event.
+CorrectAnswer: The correct answer
+Answer1: Incorrect answer
+Answer2: Incorrect answer
+PossibleAnswer3: Sometimes a third alternative incorrect answer is given
+canary_string: The first element contains the BIG Bench canary string to prevent data leackage
```

Comment:
```suggestion
- `PCTID`: The ID of the item
- `BatchID`: A ID referencing creation process. Can be ignored.
- `Premise`: The description of the base premise that is the reference event against which the hypothetical event is compared.
- `QCC`: The so called questionized counterfactual conditional is a question creating a hypothetical event and contrasting it in a query-styled natural language form to the base premise event.
- `CorrectAnswer`: The correct answer
- `Answer1`: Incorrect answer
- `Answer2`: Incorrect answer
- `PossibleAnswer3`: Sometimes a third alternative incorrect answer is given
- `canary_string`: The first element contains the BIG Bench canary string to prevent data leackage
```